src/common/in_mem_overflow_buffer_utils.cpp

```diff
@@ -27,23 +27,24 @@ void InMemOverflowBufferUtils::copyListRecursiveIfNested(const ku_list_t& src, k
     }
     assert(srcEndIdx < src.size);
     auto numElements = srcEndIdx - srcStartIdx + 1;
-    auto elementSize = Types::getDataTypeSize(*dataType.getChildType());
+    auto childType = VarListType::getChildType(&dataType);
+    auto elementSize = storage::StorageUtils::getDataTypeSize(*childType);
```

Comment:
Is using a function from `storage` module here a good idea?

Reply:
As we discussed, let's change the datatype size stuff in another pr